packages/eslint-plugin-pf-codemods/src/rules/v6/avatarReplaceBorderForIsBordered/avatar-replace-border-for-isBordered.md

```diff
@@ -0,0 +1,17 @@
+### avatar-replace-border-for-isBordered [(#9954)](https://github.com/patternfly/patternfly-react/pull/9954)
+
+border prop has been removed since light and dark borders are no longer supported. isBordered prop was added to add a border to avatar.
```

Comment:
More of a nit, but we could rephrase the message to convey that border theming is no longer handled React-side (since dark/light borders are technically supported, just not something the consumer needs to really specify at the component level). Maybe something like:

```suggestion
`border` prop has been removed from Avatar since theming is not longer handled React-side. We recommend using the `isBordered` prop instead to add a border to Avatar.
```